src/OpenTelemetry.Contrib.Instrumentation.Quartz/OpenTelemetry.Contrib.Instrumentation.Quartz.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>OpenTelemetry Quartz.NET Instrumentation</Description>
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/CODEOWNERS Could you update this to list you as an owner.

Reply:
updated